README.md

```diff
@@ -14,6 +14,6 @@ To edit source
 
 ## Using
 
-Protip: Don't
+ProTip™: Don't
```

Comment:
@shelvacu this project is a disgrace, how dare you not merge my feature!!!!!!!!!1!!!1!!!!! I will be forking ASAP

You suck